mate-session/gsm-logout-dialog.c

```diff
@@ -464,30 +464,30 @@ gsm_get_dialog (GsmDialogLogoutType type,
                 logout_dialog->default_response = GSM_LOGOUT_RESPONSE_SHUTDOWN;
 
                 if (gsm_logout_supports_system_suspend (logout_dialog)) {
-                        gtk_dialog_add_button (GTK_DIALOG (logout_dialog),
-                                               _("S_uspend"),
+                        gsm_util_dialog_add_button (GTK_DIALOG (logout_dialog),
+                                               _("S_uspend"), "system-suspend",
                                                GSM_LOGOUT_RESPONSE_SLEEP);
                 }
 
                 if (gsm_logout_supports_system_hibernate (logout_dialog)) {
-                        gtk_dialog_add_button (GTK_DIALOG (logout_dialog),
-                                               _("_Hibernate"),
+                        gsm_util_dialog_add_button (GTK_DIALOG (logout_dialog),
+                                               _("_Hibernate"), "system-hibernate",
                                                GSM_LOGOUT_RESPONSE_HIBERNATE);
                 }
 
                 if (gsm_logout_supports_reboot (logout_dialog)) {
-                        gtk_dialog_add_button (GTK_DIALOG (logout_dialog),
-                                               _("_Restart"),
+                        gsm_util_dialog_add_button (GTK_DIALOG (logout_dialog),
+                                               _("_Restart"), "gtk-refresh",
```

Comment:
`gtk-refresh` is deprecated, you can use `view-refresh` instead, all the icon names with `"gtk-..."` in the name are deprecated

Reply:
Ok for "view-refresh", updated soon, thanks.

Reply:
Done, switch to "view-refresh".